SYCL/Regression/context_with_multiple_devices.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env NEOReadDebugKeys=1 CreateMultipleRootDevices=2 %GPU_RUN_PLACEHOLDER %t.out
+// REQUIRES: level_zero && gpu
+
```

Comment:
```suggestion
// UNSUPPORTED: gpu-intel-dg1
```
Debug options above are not supported on DG1 card